lib/RequestLogger.js

```diff
@@ -222,6 +222,19 @@ class RequestLogger {
     }
 
     /**
+     *  This function creates a new RequestLogger object from a ModuleLogger
+     *  already existing in order to have the submodule field from the
+     *  ModuleLogger.
+     *
+     *	@param {ModuleLogger} ml - The base ModuleLogger
+     *	@return {RequestLogger} - The new RequestLogger created
+     *				  with the ModuleLogger
+     */
+    scope(ml) {
```

Comment:
Afair, the API we discussed was more along the lines of:
```
scope(subname) {
    return ScopedRequestLogger(this, subname);
}
```
with the ScopedRequestLogger  being very similar to the EndLogger in terms of API and capacity.

Why didn't you go that way ? (I'm feeling like you are attempting various things without attempting what we discussed. Is there any reason?)